src/rgw/rgw_pubsub.h

```diff
@@ -194,40 +194,37 @@ struct rgw_pubsub_s3_notifications {
 }*/
 
 struct rgw_pubsub_s3_record {
-  constexpr static const char* const json_type_single = "Record";
   constexpr static const char* const json_type_plural = "Records";
-  // 2.1
-  std::string eventVersion;
+  std::string eventVersion = "2.2";
   // aws:s3
-  std::string eventSource;
+  std::string eventSource = "ceph:s3";
```

Comment:
the change from "aws:s3" to "ceph:s3" is according to the comment [here](https://github.com/cloudevents/spec/blob/master/adapters/aws-s3.md) regarding "other cloud providers"